packages/components/tests/dummy/app/templates/components/form/masked-input.hbs

```diff
@@ -106,6 +106,67 @@
     {{/each}}
   {{/let}}
 
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>Copy button</Shw::Text::H3>
+
+  <Shw::Text::H4>Single line</Shw::Text::H4>
+
+  {{#let (array "base" "invalid" "readonly") as |variants|}}
+    {{#each variants as |variant|}}
+      <Shw::Grid @columns={{3}} as |SG|>
+        {{#each @model.STATES as |state|}}
+          <SG.Item
+            @label="{{capitalize variant}} / {{capitalize state}}"
+            mock-state-value={{state}}
+            mock-state-selector="input"
+          >
+            <Shw::Flex @direction="column" as |SF|>
+              <SF.Item>
+                <Hds::Form::MaskedInput::Base
+                  disabled={{if (eq variant "disabled") "disabled"}}
+                  readonly={{if (eq variant "readonly") "readonly"}}
+                  @hasCopyButton={{true}}
+                  @value="Lorem ipsum dolor"
+                  @isInvalid={{if (eq variant "invalid") true}}
+                />
+              </SF.Item>
+            </Shw::Flex>
+          </SG.Item>
+        {{/each}}
+      </Shw::Grid>
+    {{/each}}
+  {{/let}}
+
+  <Shw::Text::H4>Multiline</Shw::Text::H4>
+
+  {{#let (array "base" "invalid" "readonly") as |variants|}}
+    {{#each variants as |variant|}}
+      <Shw::Grid @columns={{3}} as |SG|>
+        {{#each @model.STATES as |state|}}
+          <SG.Item
+            @label="{{capitalize variant}} / {{capitalize state}}"
+            mock-state-value={{state}}
+            mock-state-selector="input"
```

Comment:
should this be `textarea`? (also above). I don't see the `focus` state rendered correctly in the showcase

Reply:
indeed, pushed a commit to fix the two